integration_tests/tests/assess/view_submitted_application.cy.ts

```diff
@@ -13,9 +13,10 @@ import { submittedApplicationFactory } from '../../../server/testutils/factories
 import paths from '../../../server/paths/assess'
 import Page from '../../pages/page'
 import SubmittedApplicationPage from '../../pages/assess/submittedApplicationPage'
+import { fullPersonFactory } from '../../../server/testutils/factories/person'
 
 context('Assessor views submitted application', () => {
-  const submittedApplication = submittedApplicationFactory.build()
+  // const submittedApplication = submittedApplicationFactory.build()s
```

Comment:
did you mean to leave this in as commented out, or were you intending to remove it?